diff -burN Rcs-1.05.orig/examples/access.pl Rcs-1.05/examples/access.pl
--- Rcs-1.05.orig/examples/access.pl	2004-02-16 23:09:03.634178320 +0100
+++ Rcs-1.05/examples/access.pl	2004-02-16 23:10:33.756477648 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Add users to access list.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/ci.pl Rcs-1.05/examples/ci.pl
--- Rcs-1.05.orig/examples/ci.pl	2004-02-16 23:09:03.632178624 +0100
+++ Rcs-1.05/examples/ci.pl	2004-02-16 23:10:33.787472936 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Check-in source file.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/comments2.pl Rcs-1.05/examples/comments2.pl
--- Rcs-1.05.orig/examples/comments2.pl	2004-02-16 23:09:03.636178016 +0100
+++ Rcs-1.05/examples/comments2.pl	2004-02-16 23:10:33.800470960 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Access comments hash
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/comments.pl Rcs-1.05/examples/comments.pl
--- Rcs-1.05.orig/examples/comments.pl	2004-02-16 23:09:03.638177712 +0100
+++ Rcs-1.05/examples/comments.pl	2004-02-16 23:10:33.820467920 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Access comments hash
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/co.pl Rcs-1.05/examples/co.pl
--- Rcs-1.05.orig/examples/co.pl	2004-02-16 23:09:03.632178624 +0100
+++ Rcs-1.05/examples/co.pl	2004-02-16 23:10:33.830466400 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Check-out source file.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/daterev.pl Rcs-1.05/examples/daterev.pl
--- Rcs-1.05.orig/examples/daterev.pl	2004-02-16 23:09:03.636178016 +0100
+++ Rcs-1.05/examples/daterev.pl	2004-02-16 23:10:33.850463360 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Test daterev method
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/dates.pl Rcs-1.05/examples/dates.pl
--- Rcs-1.05.orig/examples/dates.pl	2004-02-16 23:09:03.632178624 +0100
+++ Rcs-1.05/examples/dates.pl	2004-02-16 23:10:33.861461688 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Access dates hash
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/parse.pl Rcs-1.05/examples/parse.pl
--- Rcs-1.05.orig/examples/parse.pl	2004-02-16 23:09:03.635178168 +0100
+++ Rcs-1.05/examples/parse.pl	2004-02-16 23:10:33.871460168 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Parse RCS archive file.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/rcsclean.pl Rcs-1.05/examples/rcsclean.pl
--- Rcs-1.05.orig/examples/rcsclean.pl	2004-02-16 23:09:03.633178472 +0100
+++ Rcs-1.05/examples/rcsclean.pl	2004-02-16 23:10:33.889457432 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # rcsclean utility
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/rcsdiff.pl Rcs-1.05/examples/rcsdiff.pl
--- Rcs-1.05.orig/examples/rcsdiff.pl	2004-02-16 23:09:03.634178320 +0100
+++ Rcs-1.05/examples/rcsdiff.pl	2004-02-16 23:10:33.899455912 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Use rcsdiff utility.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/revdate.pl Rcs-1.05/examples/revdate.pl
--- Rcs-1.05.orig/examples/revdate.pl	2004-02-16 23:09:03.632178624 +0100
+++ Rcs-1.05/examples/revdate.pl	2004-02-16 23:10:33.909454392 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Get revision date
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/rlog.pl Rcs-1.05/examples/rlog.pl
--- Rcs-1.05.orig/examples/rlog.pl	2004-02-16 23:09:03.633178472 +0100
+++ Rcs-1.05/examples/rlog.pl	2004-02-16 23:10:33.959446792 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Use rlog utility.
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/symbols.pl Rcs-1.05/examples/symbols.pl
--- Rcs-1.05.orig/examples/symbols.pl	2004-02-16 23:09:03.633178472 +0100
+++ Rcs-1.05/examples/symbols.pl	2004-02-16 23:10:33.968445424 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Test symrev method
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/symrev.pl Rcs-1.05/examples/symrev.pl
--- Rcs-1.05.orig/examples/symrev.pl	2004-02-16 23:09:03.636178016 +0100
+++ Rcs-1.05/examples/symrev.pl	2004-02-16 23:10:33.978443904 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Test symrev method
 #------------------------------------------
diff -burN Rcs-1.05.orig/examples/test1.pl Rcs-1.05/examples/test1.pl
--- Rcs-1.05.orig/examples/test1.pl	2004-02-16 23:09:03.634178320 +0100
+++ Rcs-1.05/examples/test1.pl	2004-02-16 23:10:33.988442384 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use lib '.';
diff -burN Rcs-1.05.orig/examples/test2.pl Rcs-1.05/examples/test2.pl
--- Rcs-1.05.orig/examples/test2.pl	2004-02-16 23:09:03.635178168 +0100
+++ Rcs-1.05/examples/test2.pl	2004-02-16 23:10:33.997441016 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use lib '.';
diff -burN Rcs-1.05.orig/examples/test3.pl Rcs-1.05/examples/test3.pl
--- Rcs-1.05.orig/examples/test3.pl	2004-02-16 23:09:03.635178168 +0100
+++ Rcs-1.05/examples/test3.pl	2004-02-16 23:10:34.007439496 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use lib '.';
diff -burN Rcs-1.05.orig/examples/unlock.pl Rcs-1.05/examples/unlock.pl
--- Rcs-1.05.orig/examples/unlock.pl	2004-02-16 23:09:03.637177864 +0100
+++ Rcs-1.05/examples/unlock.pl	2004-02-16 23:10:34.017437976 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #------------------------------------------
 # Unlock RCS file
 #------------------------------------------
